ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -55,6 +55,10 @@ let prototyped_of_field = function
       Some "23.9.0"
   | "pool", "telemetry_uuid" ->
       Some "23.9.0"
+  | "pool", "ext_auth_max_threads" ->
+      Some "23.25.0-next"
```

Comment:
We have tag v23.26.0 already, so I think you need to fetch the tags and run this again.

Reply:
```
git describe
v23.26.0-7-gd90369deb
```
something goes wrong on why it picks up 25 instead

Reply:
This is due to the dune cache, it needs to be told that 'gen_lifecycle' depends on something external (i.e. 'the universe'), otherwise it'll use a cached version that still uses the old version number.
I pushed a commit that makes the dune change and now it prints the right version.